addons/turbowarp-redirect/addon.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "Turbowarp redirect",
```

```suggestion
  "name": "TurboWarp redirect",
```
TurboWarp is a name.